scripts/scil_volume_pairwise_comparison.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Evaluate pair-wise similarity measures of masks and atlas.
+All volumes must be in the same space.
+
+For the voxel representation, the computed similarity measures are:
+    adjacency_voxels, dice_voxels, volume_overlap, volume_overreach.
+
+This can work for BET mask, WMPARC, bundle label maps. The datatype of the
+input volumes must be uint8 (mask) or uint16 (label map and atlas).
```

Comment:
Could there be a --force_input_type option, if you don't want to save your file on disk with other input type?

Reply:
Almost all of the scilpy scripts enforce labels as uint16 and mask as uint8, I think we want to force user to change their data type.